modules/site-health/admin-site-health.php

```diff
@@ -0,0 +1,272 @@
+		$options = get_option( 'polylang' );
+		$fields = array();
+		foreach ( $options as $key => $value ) {
+			if ( in_array(
+				$key,
+				$this->exclude_options_key()
+			)
+				) {
+				continue;
+			}
+			if ( ! is_array( $value ) ) {
+				if ( false === $value ) {
+					$value = '0';
+				}
+				$fields[ $key ]['label']   = $key;
+				$fields[ $key ]['value']   = $value;
+			} else {
+				if ( empty( $value ) ) {
+					$fields[ $key ]['label']   = $key;
+					$fields[ $key ]['value']   = '0';
+				} else {
+					switch ( $key ) {
+						case 'post_types':
+							$fields[ $key ]['label'] = $key;
+							$fields[ $key ]['value'] = implode( ', ', $this->model->get_translated_post_types() );
+							break;
+						case 'taxonomies':
+							$fields[ $key ]['label'] = $key;
+							$fields[ $key ]['value'] = implode(
+								', ',
+								$this->model->get_translated_taxonomies()
+							);
+							break;
+						case 'nav_menus':
+							$current_theme = get_stylesheet();
+							foreach ( $value[ $current_theme ] as $location => $lang ) {
+								// translators: placeholder is the menu location name
+								$fields[ $location ]['label'] = sprintf( __( 'Menu: %s', 'polylang' ), $location );
+								array_walk(
+									$lang,
+									function ( &$value, $key ) {
+										$value = "$key:$value";
+									}
+								);
+								$fields[ $location ]['value'] .= implode( ' | ', $lang );
+							}
+							break;
+						case 'media':
+							array_walk(
+								$value,
+								function ( &$value, $key ) {
+									$value = "$key: $value";
+								}
+							);
+							$fields[ $key ]['label'] = '';
+							$fields[ $key ]['value'] .= implode( ',', $value );
+							break;
+						default:
+							$fields[ $key ]['label']   = $key;
+							$fields[ $key ]['value']   = implode( ', ', $value );
+							break;
+					}
+				}
+			}
+		}
+
+		$debug_info['polylang'] = array(
+			'label'    => __( 'Polylang Options', 'polylang' ),
+			'fields' => $fields,
+		);
+
+		return $debug_info;
+	}
+
+	/**
+	 * Add Polylang Languages settings to Site Health Informations tab.
+	 *
+	 * @param array $debug_info array options to display.
+	 * @return array
+	 * @since   2.8
+	 */
+	public function info_languages( $debug_info ) {
+		$languages = PLL()->model->get_languages_list();
+		$fields = array();
+		foreach ( $languages as $language ) {
+			foreach ( $language as $key => $value ) {
+				if ( empty( $value ) ) {
+					$value = '0';
+				}
+
+				if ( in_array(
+					$key,
+					$this->exclude_lang_key()
+				)
+				) {
+					continue;
+				}
+
+				$fields[ $key ]['label'] = $key;
+				$fields[ $key ]['value'] = $value;
+
+				if ( 'term_group' === $key ) {
+					$fields[ $key ]['label'] = _x( 'order', 'Order of the language in the language switcher', 'polylang' );
+				}
+			}
+			$flag = $this->get_flag( $language );
+
+			$lang_name = sanitize_title( $language->name );
+			$debug_info[ $lang_name ] = array(
```

Comment:
From [WordPress' Site Health Check documentation](https://make.wordpress.org/core/2019/04/25/site-health-check-in-5-2/):

> Each new section that’s added should use your plugin or theme slug to avoid name collisions. All core entries are prefixed with wp-.

We should have a way to distinguish our languages from those of another multilingual plugin, otherwise, they may be overridden.